### PR TITLE
Ms reorganize components

### PR DESCRIPTION
**Pull request by NicoTuxx**
## Checklist

Before you submit this pull request, please make sure to:

- [x] Keep changes in the pull request as small as possible
   
- [x] Ensure the commit history is sanitized
   
- [x] Give a meaningful title to your PR
- [x] Describe your changes
   
- [x] Link any related issue in the description
   
- [x] Link any dependent pull request in the description

If this PR adds a new feature or fixes a bug:

- [x] I have added or updated relevant tests

If this PR modifies installation or updates packages:

- [x] I have checked that installation works on all platforms (Windows, Linux, Mac ...)
    

- [x] I have updated related documentations (installation commands, fix to specific problems ...)

If this PR is related to GUI:

- [x] I have checked that GUIs affected by my changes are working properly.

If this PR affects the user:

- [ ] I have created a news fragment
   
- [ ] I have updated the user documentation

### CODE REVIEW

File: client/src/views/home/HomePage.vue [hunk 1]
```diff
@@ -583,3 +583,5 @@ async function openSettingsModal(): Promise<void> {
   }
 }
 </style>
+@/components/core/ms-select/MsSelectOption
```
Comment (Max-7) [inline client/src/views/home/HomePage.vue:586]: ??

File: client/src/views/workspaces/WorkspacesPage.vue [hunk 2]
```diff
@@ -301,3 +301,5 @@ ion-item::part(native) {
   display: flex;
 }
 </style>
+@/components/core/ms-select/MsSelectOption
```
Comment (Max-7) [inline client/src/views/workspaces/WorkspacesPage.vue:304]: ??

File: client/src/components/core/ms-select/MsSelect.vue [hunk 3]
```diff
@@ -88,3 +88,4 @@ async function onDidDismissPopover(popover: any): Promise<void> {
   }
 }
 </style>
+@/components/core/ms-select/MsSelectOption
```
Comment (Max-7) [inline client/src/components/core/ms-select/MsSelect.vue:91]: ??

File: client/src/components/workspaces/WorkspaceUserRole.vue [hunk 4]
```diff
@@ -77,3 +77,5 @@ function getRoleFromString(role: string): WorkspaceRole | null {
   }
 }
 </style>
+@/components/core/ms-select/MsSelectOption
```
Comment (Max-7) [inline client/src/components/workspaces/WorkspaceUserRole.vue:80]: ??

**Review by Max-7 — CHANGES_REQUESTED** [review]

**Review by Max-7 — APPROVED** [review]

**Review by fabienSvtr — APPROVED** [review]
Most of css class and TypeScript variables of components have been changed, don't think that was necessary but it works so that's fine.

**Pull request merged**